tests/src/test/java/de/sovity/edc/e2e/Ms8ConnectorMigrationTest.java

```diff
@@ -124,15 +123,6 @@ void testMs8DataOffer_Properties() {
             softly.assertThat(asset.getPublisherHomepage()).isEqualTo("https://publisher");
             softly.assertThat(asset.getTransportMode()).isEqualTo("Rail");
             softly.assertThat(asset.getVersion()).isEqualTo("1.0");
-            softly.assertThat(asset.getAdditionalProperties()).isEqualTo(Map.of(
-                    "http://unknown/usecase", "my-use-case",
-                    "http://unknown/custom-prop-1", "1",
-                    "http://custom-prop-2", "2",
-                    "https://custom-prop-3", "3"
-            ));
```

Comment:
Should we add a customJsonString assertion to replace this? and keep the fields in the migration file?